pkg/backend/openshiftcluster/create.go

```diff
@@ -31,17 +31,49 @@ import (
 
 	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/install"
+	"github.com/Azure/ARO-RP/pkg/util/pullsecret"
 	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 	"github.com/Azure/ARO-RP/pkg/util/subnet"
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
+func getACRRegistryProfile(oc *api.OpenShiftCluster) *api.RegistryProfile {
+	for ix, rp := range oc.Properties.RegistryProfiles {
+		if rp.Type == api.RegistryTypeACR {
+			return &oc.Properties.RegistryProfiles[ix]
+		}
+	}
+	return nil
+}
+
 func (m *Manager) Create(ctx context.Context) error {
 	var err error
 
 	resourceGroup := stringutils.LastTokenByte(m.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, '/')
 
+	var acrRP *api.RegistryProfile
+	if m.env.ACRResourceID() != "" {
```

Comment:
Some comment would be great to state in what scenario ACRResourceID can and can't be empty

Reply:
ok, (basically dev).